git-subsplit.sh

```diff
@@ -334,7 +334,6 @@ subsplit_update()
 
 	if [ -n "$VERBOSE" ];
 	then
-		echo "${DEBUG} git fetch -q origin"
```

Comment:
You removed the duplicate in the echo but not on line 330 above where it is done for real. It is unclear to me if I actually need both w/ and w/o tags. If you know for sure we don't need `git fetch` bare you can comment out the actual line above, too, if you want to. I always thought `--tags` were optional and if you *did* get tags it was tags only. Though that might just be `git push` behavior. :)